tools/wptrunner/wptrunner/executors/executorchrome.py

```diff
@@ -189,8 +190,21 @@ def pdf_to_png(self, pdf_base64, ranges):
             self.webdriver.window_handle = handle
 
 
+class ChromeDriverFedCMProtocolPart(WebDriverFedCMProtocolPart):
+    def setup(self):
+        self.webdriver = self.parent.webdriver
+        # Company prefix to apply to vendor-specific WebDriver extension commands.
+        self.fedcm_company_prefix = "goog"
+
+
+    def confirm_idp_signin(self):
+        return self.webdriver.send_session_command("POST",
+                                                   self.fedcm_company_prefix + "/fedcm/confirmidpsignin")
+
+
 class ChromeDriverProtocol(WebDriverProtocol):
     implements = [
+        ChromeDriverFedCMProtocolPart,
         ChromeDriverPrintProtocolPart,
         ChromeDriverTestharnessProtocolPart,
         *(part for part in WebDriverProtocol.implements
```

Comment:
To fix the name conflict on CI, you'll want to also exclude the `WebDriverFedCMProtocolPart` "inherited" from `WebDriverProtocol.implements` (like we're doing here with `ChromeDriverTestharnessProtocolPart`).